code/modules/urist/items/clothstuff.dm

```diff
@@ -94,3 +94,60 @@
 				else
 					to_chat(user, "<span class='warning'>\The [src] is used up, but there are more wounds to treat on \the [affecting.name].</span>")
 			use(used)
+
+/obj/item/loom
+	name = "table loom"
+	desc = "A loom small enought to only take up a table instead of the whole floor."
+	icon = 'icons/urist/items/wood.dmi'
+	icon_state = "loom"
+
+/datum/reagent/cottonfiber
+	name = "cotton fiber"
+	description = "A mass of cotton fibers."
+	taste_description = "cotton"
+	reagent_state = SOLID
+	color = "#ffffff"
+
+/datum/seed/cotton
+	name = "cotton"
+	seed_name = "cotton"
+	display_name = "cotton plant"
+	chems = list(/datum/reagent/cottonfiber = list(6,1))
+
+/datum/seed/cotton/New()
+	..()
+	set_trait(TRAIT_MATURATION,1)
+	set_trait(TRAIT_PRODUCTION,1)
+	set_trait(TRAIT_YIELD,7)
+	set_trait(TRAIT_POTENCY,10)
+//	set_trait(TRAIT_PRODUCT_ICON,"cotton")
```

Comment:
commented out for the time being because for unknown reasons, this is not recognized as an icon even though its in the DMI. and the potato icon would be an ok substitute but it isnt showing up as white.